src/scenarios/shared/util.py

```diff
@@ -34,6 +34,10 @@ def builtexe(exename: str):
     'gets binary path'
     return os.path.join(const.BINDIR, '%s%s' % (exename, extension()))
 
+def appfolder(projname: str):
+    'gets path for callign dotnet run'
+    return os.path.join(const.APPDIR, '%s.csproj' % projname)
```

Comment:
should this include the project extension? We have some VB floating around etc.

Reply:
Done, had to solve this by adding a new test trait, but it is pretty clean.